ruby-on-rails-v6/README.md

```diff
@@ -0,0 +1,17 @@
+# README
```

Comment:
Can you please make readme file similar to https://github.com/tradingview/charting-library-examples/tree/master/ruby-on-rails?

Reply:
yes